### PR TITLE
Endpoint remove ssm parameter

### PR DESCRIPTION
**Pull request by Yuuki77**
Part of #644

now add the endpoint to delete ssm parameter from aws
path is `v1/districts/:district_name/ssm_parameter`.

You need to pass 
```ruby
   name: name, # required
```

### CODE REVIEW

**Comment by degicat** [comment]
@davidsiaw can you help us review this PR, please?

File: spec/services/ssm_parameters_spec.rb [hunk 1]
```diff
@@ -13,9 +13,17 @@
...
       response = described_class.new(district, name).put_parameter(parameter_value)
       expect(response.version).to eq 0
     end
   end
+
+  describe "#delete_parameter" do
+    it "delete ssm parameter" do
+      response = described_class.new(district, name).delete_parameter
+      expect(response.deleted_parameters).to eq []
+      expect(response.invalid_parameters).to eq []
```
Comment (davidsiaw) [inline spec/services/ssm_parameters_spec.rb:26]: We should also check here that we are sending out a delete request to AWS. (just mocking AWS will do, we don't have to really send it)

**Review by davidsiaw — CHANGES_REQUESTED** [review]
Looks great! Thank you! Just a small thing

File: spec/services/ssm_parameters_spec.rb (hunk 1 above)
Comment (Yuuki77) [reply spec/services/ssm_parameters_spec.rb:29]: @davidsiaw Thank you for the reviews!
updated here https://github.com/degica/barcelona/pull/679/commits/a29189515cec43ee688f371c151b6fb13b1be68c
currently the request is already mocked so it should be ok.

**Review by davidsiaw — APPROVED** [review]
LGTM

**Pull request merged**